pytest-selenium/tests/conftest.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+import pytest
+
+# Window resolutions
+DESKTOP = (1080, 1920)
+MOBILE = (414, 738)
+
+
+@pytest.fixture(
+    scope="function", params=[DESKTOP, MOBILE], ids=["Resolution: 1080x1920", "Resolution: 414x738"]
```

Comment:
resolutions duplicated here and above?

Reply:
The ids are mainly for display when the tests are running. It's not actually duplicating them but rather adding a label.

Reply:
Example:
![image](https://user-images.githubusercontent.com/8730430/51543662-b1f7b380-1e23-11e9-8de3-4b16717f4015.png)

Reply:
but why not use the constants in these strings?

Reply:
I did